application/src/main/java/org/togetherjava/tjbot/commands/utils/StringDistances.java

```diff
@@ -65,6 +65,14 @@ public static <S extends CharSequence> Optional<S> autocomplete(CharSequence pre
      */
     public static Collection<String> closeMatches(CharSequence prefix,
             Collection<String> candidates, int limit) {
+        if (prefix.isEmpty()) {
```

Comment:
if the prefix is empty, the returned list should be empty, since any suggestion can only be a bad suggestion.

Reply:
yes, but i dont like to see the no auto-complete suggestions even before i did anything
![image](https://user-images.githubusercontent.com/73871477/194723825-c2569e51-ab4d-4157-b235-a569e2fc99db.png)

Reply:
lol whats that. cant it just tell discord that it has nothing and discord doesnt show anything extra then? like without autocomplete? @Tais993

Reply:
no

Reply:
that would be nice tho

Reply:
anyways, ur doing sorted() again instead of k-sort. thats the same mistake than before, a performance-problem. use a PQ.